libr/debug/debug.c

```diff
@@ -343,6 +343,11 @@ R_API int r_debug_select(RDebug *dbg, int pid, int tid) {
 	if (dbg->h && dbg->h->select && !dbg->h->select (pid, tid))
 		return false;
 
+	{
+		char buf[32];
+		sprintf (buf, "pid %d", pid);
+		r_io_system (dbg->iob.io, buf);
```

Comment:
It's better but 1) it would leak memory, no? 2) it's still ugly in that we are passing a string command to IO.

What would be better is to have some API like ```dbg->iob.io->select_pid(pid)``` ...